### DISCUSSION TITLE
space inside import braces

### DISCUSSION THREAD

**Discussion by renegat4** [neoclide/coc.nvim#4151] [Q&A] [answered]
I am using vim 9.0 and coc.nvim 0.0.82 to write typescript.

Some time ago a managed to configure my workspace to add spaces inside automatically added imports.

```ts
import { something } from 'module';
```

But some days ago i updated coc.nvim an now imports are like this again:

```ts
import {something} from 'module';
```

And i cant figure out which configuration i changed to add the whitespace. To be honest i dont even know if it was coc-configuration or some typescript-configuration or ... But since it changed after the coc.nvim update i presume it has to do with coc.nvim.

**Comment by chemzqm** [comment, marked as answer]
Checkout https://github.com/neoclide/coc-tsserver#configuration-options, one possible reason is you've installed coc-prettier to format the buffer instead of coc-tssserver
